libtiledbsoma/test/unit_soma_sparse_ndarray.cc

```diff
@@ -148,6 +147,12 @@ TEST_CASE("SOMASparseNDArray: basic", "[SOMASparseNDArray]") {
             REQUIRE_THROWS(soma_sparse->resize(new_shape));
             // Now set the shape
             soma_sparse->upgrade_shape(new_shape);
+            soma_sparse->close();
+
+            soma_sparse->open(OpenMode::read);
+            soma_sparse->close();
+
+            soma_sparse->open(OpenMode::write);
```

Comment:
What does this do or check for?

Reply:
Bad rebase I think --
```
REQUIRE(soma_sparse->has_current_domain());
```
-- 2nd commit coming up -- thank you! :)

https://github.com/single-cell-data/TileDB-SOMA/pull/3068/commits/a2bb0d08363bd87b37e04e4082a43132178b6a52